template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/WebApi-CSharp/Areas/IdentityService/Views/IdentityService/LoggedOut.cshtml

```diff
@@ -0,0 +1,5 @@
+@{
+    ViewData["Title"] = "Sign Out";
```

Comment:
Yeah that's basically how identity is with SignInManager/SignIn/SignOut methods, but UserManager describes the information about how to sign in as UserLogins